src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -60,6 +61,7 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
 register_cli_argument('appservice web create', 'name', options_list=('--name', '-n'), help='name of the new webapp')
 register_cli_argument('appservice web create', 'plan', options_list=('--plan', '-p'), completer=get_resource_name_completion_list('Microsoft.Web/serverFarms'),
                       help="name or resource id of the app service plan. Use 'appservice plan create' to get one")
+register_cli_argument('appservice web browse', 'start_trace', action='store_true', help='start live tracing')
```

Comment:
Getting the `-l` alias would definitely be great. That said, I don't know if we need to worry about Kudu endpoints/naming terminology, as opposed to choosing to optimizing the name and description around making the CLI experience as awesome as possible.

I'd be in favor of naming the `zipped file log` parameters something like `--log-file`, and giving the log stream args priority in naming, since those are the workflow that we currently lead with.

Since the `browse` command won't be exposing both the zip and stream solutions, it also seems fine to just name this arg `--logs`, as long as the description makes it clear what it does `Enable viewing the log stream immediately after launching the web app` (or something similar).